utils/cstyle

```diff
@@ -332,12 +332,12 @@ line: while (<$filehandle>) {
 		    s/\t+/' ' x (length($&) * 8 - length($`) % 8)/e;
 		if (length($eline) > 80) {
 			# allow long line if it is user visible string
-			# find if line start from " and ends
+			# find if line start from " or L"and ends
```

Comment:
please do it in a separate commit, I might need to cherry-pick to other branches